.github/workflows/release_pypi.yml

```diff
@@ -85,4 +86,4 @@ jobs:
       - name: Build documentation
         run: |
           version=$(git describe --tags --abbrev=0)
-          mike deploy -p -u $version latest
+          mike deploy -p -u $version latest --allow-empty
```

Comment:
I believe that otherwise, a new commit cannot be created if we don't explicitly change the docs files, but I might be wrong. I guess it doesn't hurt to have this setting active though.